projects/media-viewer/src/lib/annotations/models/tags.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface TagItemModel {
+export interface TagsModel {
```

Comment:
why not just call it Tags

Reply:
dunno maybe because Tags dos not mean is a Model - ti can be an array